nexus_allowlist/nexus.py

```diff
@@ -123,6 +126,8 @@ def create_proxy_repository(
         }
         payload["name"] = name
         payload["proxy"]["remoteUrl"] = remote_url
+        if repo_type == RepositoryType.APT:
+            payload["apt"] = {"distribution": APT_DISTRO, "flat": False}
```

Comment:
This looks problematic. I think what we probably want to do is pass an object with the repositories actual parameters.